### ISSUE TITLE
dotnet missing versions of GLIBCXX (CentOS)

### ISSUE THREAD

**Issue by RicardoRazz** [dotnet/cli#3298]
Hi.
I was trying to run a DNX application on my CentOS 6.8 server but I'm not able to run anything at all. Before installing the RC2 I've installed all the linux dependencies.
This is what I get when running any dotnet command (launching):

**[root@user lib64]#** dotnet --info
dotnet: /usr/lib64/libstdc++.so.6: version `GLIBCXX_3.4.18' not found (required by dotnet)

dotnet: /usr/lib64/libstdc++.so.6: version `CXXABI_1.3.5' not found (required by dotnet)

dotnet: /usr/lib64/libstdc++.so.6: version `GLIBCXX_3.4.14' not found (required by dotnet)

dotnet: /usr/lib64/libstdc++.so.6: version `GLIBCXX_3.4.15' not found (required by dotnet)

These are the lines contained in the libstdc++.so.6:

**[root@user lib64]#** strings libstdc++.so.6|grep GLIBC
GLIBCXX_3.4
GLIBCXX_3.4.1
GLIBCXX_3.4.2
GLIBCXX_3.4.3
GLIBCXX_3.4.4
GLIBCXX_3.4.5
GLIBCXX_3.4.6
GLIBCXX_3.4.7
GLIBCXX_3.4.8
GLIBCXX_3.4.9
GLIBCXX_3.4.10
GLIBCXX_3.4.11
GLIBCXX_3.4.12
GLIBCXX_3.4.13
GLIBC_2.2.5
GLIBC_2.3
GLIBC_2.4
GLIBC_2.3.2
GLIBCXX_FORCE_NEW
GLIBCXX_DEBUG_MESSAGE_LENGTH

Does anyone know how to solve this?

## Steps to reproduce

- Basically just call the dotnet command.

## Expected  behavior

- Run the application.

## Actual behavior

- CLI does not run.

**Comment by piotrpMSFT** [comment]
We started CentOS support with 7.1 and don't have plans to pursue earlier versions. I do hope that folks in the community might comment on this issue with a solution, however!

**Comment by avinb** [comment]
I am facing this issue on Oracle Linux 6.5 .   

Kernel 2.6.32-642 .

**Comment by nbnbnb** [comment]
I have the same error on CentOS 6.5 x64

dotnet: /usr/lib64/libstdc++.so.6: version `GLIBCXX_3.4.18' not found (required by dotnet)
dotnet: /usr/lib64/libstdc++.so.6: version `CXXABI_1.3.5' not found (required by dotnet)
dotnet: /usr/lib64/libstdc++.so.6: version `GLIBCXX_3.4.14' not found (required by dotnet)
dotnet: /usr/lib64/libstdc++.so.6: version `GLIBCXX_3.4.15' not found (required by dotnet)

**Comment by moliveto** [comment]
dotnet: /usr/lib64/libstdc++.so.6: version `GLIBCXX_3.4.18' not found (required by dotnet)

**Comment by moliveto** [comment]
centos 6.9

dotnet: /usr/lib64/libstdc++.so.6: version `GLIBCXX_3.4.18' not found (required by dotnet)
dotnet: /usr/lib64/libstdc++.so.6: version `CXXABI_1.3.5' not found (required by dotnet)
dotnet: /usr/lib64/libstdc++.so.6: version `GLIBCXX_3.4.14' not found (required by dotnet)
dotnet: /usr/lib64/libstdc++.so.6: version `GLIBCXX_3.4.15' not found (required by dotnet)

**Comment by nitrat7** [comment]
centos 6.8 (Elemental)


dotnet: /usr/lib64/libstdc++.so.6: version `GLIBCXX_3.4.18' not found (required by dotnet)
dotnet: /usr/lib64/libstdc++.so.6: version `CXXABI_1.3.5' not found (required by dotnet)
dotnet: /usr/lib64/libstdc++.so.6: version `GLIBCXX_3.4.14' not found (required by dotnet)
dotnet: /usr/lib64/libstdc++.so.6: version `GLIBCXX_3.4.15' not found (required by dotnet)
